groclient/lib.py

```diff
@@ -423,7 +424,7 @@ def rank_series_by_source(access_token, api_host, selections_list):
     for series_key, series_by_source_id in series_map.items():
         try:
             series_without_source = {
-                type_id: int(series_key.split('.')[idx])
+                type_id: ast.literal_eval(series_key.split('.')[idx])
                 for idx, type_id in enumerate(DATA_SERIES_UNIQUE_TYPES_ID)
                 if type_id != 'source_id' and series_key.split('.')[idx] != 'None'
             }
```

Comment:
Is it necessary for this piece of code to be within the `try`? Perhaps we can move it outside? Based on the `# empty response` comment, my guess is that the `try` is meant to catch errors from `get_source_ranking`.

If we move it out, we probably would have caught this bug via tests.

Reply:
That's a good point - I just removed it completely. i feel it's just misleading to return empty result silently